doc/source/comparison_with_r.rst

```diff
@@ -530,6 +530,103 @@ For more details and examples see :ref:`categorical introduction <categorical>`
 :ref:`differences to R's factor <categorical.rfactor>`.
 
 
+String Processing
+-----------------
+
+Length
+~~~~~~
+
+R determines the length of a character string with the ``nchar`` function. 
+``nchar`` includes leading and trailing blanks.  Use ``nchar`` and ``trimws`` 
+to exclude leading and trailing blanks. 
+
+.. code-block:: none
+
+   df <- data.frame(color = c('red', ' blue', 'green ', ' yellow '))
+   nchar(as.character(df$color))
+   nchar(trimws(as.character(df$color)))
```

Comment:
would be nice to show output here (IOW run the R code and show the output too if you can)